var/spack/repos/builtin/packages/coin3d/package.py

```diff
@@ -7,17 +7,24 @@
 from spack.pkg.builtin.boost import Boost
 
 
-class Coin3d(AutotoolsPackage):
+class Coin3d(AutotoolsPackage, CMakePackage):
     """Coin is an OpenGL-based, 3D graphics library that has its roots in the
     Open Inventor 2.1 API, which Coin still is compatible with."""
 
     homepage = "https://github.com/coin3d/coin"
-    url = "https://github.com/coin3d/coin/archive/Coin-4.0.0.tar.gz"
+    url = "https://github.com/coin3d/coin/releases/download/Coin-4.0.0/coin-4.0.0-src.tar.gz"
```

Comment:
This URL doesn't work for older versions. Does the archive file not work? If not, we might have to add a `url_for_version` to change the URL.

Reply:
fixed